### DISCUSSION TITLE
Update to PIXI v6

### DISCUSSION THREAD

**Discussion by arthuro555** [4ian/GDevelop#2360] [Archive]
Pixi v6 is about to come out, two RCs have already been published. As far as I see, the changes are primarily bug fixes and they made the typescript typing better (which is good since we just moved to TS). I think this update would be interesting as
- It fixes bugs
- It has better typescript support
- It is compatible with pixi-projection unlike our current pixi version.

**Comment by 4ian** [comment]
Upgrading is fine when we can, I think we'll need at least an upgrade for the BitmapText as there were bugfixes for it in the v5 branch.
We can either upgrade to latest v5 or jump the ship and upgrade to v6 if it's mostly bug fixes :)

**Comment by Bouh** [comment]
I waiting for V6.
For the bitmap extension I work with 5.3.7 on my local branch it's enough for me.

**Comment by arthuro555** [comment]
This can be closed, we already switched to v6

**Reply by Bouh** [reply]
[V6.1.2 ](https://github.com/pixijs/pixijs/releases/tag/v6.1.2) just released out 👀 and the version just before forward fix for rounding values.
Hopping for a better cut edge on sprites, tiling, and a better display in general 🤞


Edit: Discussion cannot be closed, it is just Locked, or for the Questions we can mark as Solved.

**Reply by arthuro555** [reply]
I know they cannot be "closed" but by that I meant locking it.
